test/sharness/t0040-add-and-cat.sh

```diff
@@ -156,6 +156,15 @@ test_add_cat_file() {
     test_cmp expected actual
   '
 
+  test_expect_success "ipfs add --chunker rabin-36-512-1024 succeeds" '
+    ipfs add -Q --chunker rabin-36-512-1024 mountdir/hello.txt
+  '
+
+  test_expect_code 127 "ipfs add --chunker rabin-12-512-1024 failed" '
```

Comment:
Ah, sorry, my comment was confusing. This needs to be:

```
test_expect_success "...." '
    test_expect_code 127 ...
'
```

(or something like that).

Reply:
(fixed)